dockerfiles/parity-ci-linux/README.md

```diff
@@ -1,6 +1,6 @@
 # Parity-ci-linux
 
-Docker image based on [Ubuntu:xenial](https://hub.docker.com/_/ubuntu).
```

Comment:
Let's return parity-eth CI to xenial unless we are sure the compiled binary will work on xenial.

Reply:
I agree, for parity-ethereum, we will leave this version for now.